docs/documentation/platform/pki/private-ca.mdx

```diff
@@ -296,4 +327,10 @@ In the following steps, we explore how to renew a CA certificate via same key pa
     At the moment, Infisical only supports CA renewal via same key pair. We
     anticipate supporting CA renewal via new key pair in the coming month.
   </Accordion>
+  <Accordion title="Does Infisical support chaining an Intermediate CA to an external Root CA?">
```

Comment:
maybe let's just make it generic as in "external CA"? 

As per my understanding, it's not only limited to chaining to root external CAs ?

Reply:
Yes will update this in separate PR